src/test/java/no/sikt/nva/scrapers/DublinCoreScraperTest.java

```diff
@@ -1377,6 +1377,20 @@ var record = dcScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(
         assertThat(record.getEntityDescription().getLanguage().getNva().toString(), containsString("sme"));
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {"Book", "Textbook", "Book of abstracts"})
+    void shouldLookUpSeriesInChannelRegisterWhenPublicationIsBookWithPartOfSeries(String type){
```

Comment:
What behavior is this test checking? The name is a bit confusing to me.

Reply:
It looks up series name in csv file and creates series with identifier from csv.

Reply:
Adding following display name to the test:

```
    @DisplayName("When brage record has type Book, Textbook or Book of abstract, and " +
                 "publication has partOfSeries field which is present, looking up for series name" +
                 "in channel register csv file and if we get a match we are setting series pid in publication context")
```

Reply:
Ok, that makes sense, but I think you'll have to explain this in more detail on Monday ;-)
It's not clear to me how much of this test is mocked and how it would look with real data. What would it look like if parsing a `TextBook` that is not part of a series?

Reply:
We can take a look at it on Monday. We do not mock anything in here :)
If publication does not have partOfSeries -> we do nothing with part of series, same behaviour as before.